src/components/policy/policy_external/include/policy/policy_helper.h

```diff
@@ -178,6 +178,14 @@ struct CheckAppPolicy {
    */
   void AddResult(const std::string& app_id, PermissionsCheckResult result);
   /**
+   * @brief Adds result of check of applications related to current application
```

Comment:
@AKalinich-Luxoft please describe this function better. It is not clear what does in means - related application to application.

Reply:
@LuxoftAKutsan description was updated in https://github.com/smartdevicelink/sdl_core/pull/1779/commits/c40c209107a48effdaf494913f9e62abd9d92334